### PR TITLE
Added missing case in CPU line drawing algorithm

### PR DESCRIPTION
**Pull request by davidpiuva**
Added the case for when Y changes more than X for each drawn pixel.

### CODE REVIEW

**Comment by davidpiuva** [comment]
@emilwestergren Peer review?

File: source/draw/RasterImage.ooc [hunk 1]
```diff
@@ -66,17 +66,32 @@ RasterImage: abstract class extends Image {
...
+				endX := start x maximum(end x)
+				for (x in startX .. endX + 1) {
+					idealY := slope * (x - start x) + start y
+					floor := idealY floor()
+					weight := (idealY - floor) abs()
+					pen setAlpha(originalAlpha * (1.0f - weight))
+					this _drawPoint(x, floor, pen)
+					pen setAlpha(originalAlpha * weight)
+					this _drawPoint(x, floor + 1, pen)
+				}
```
Comment (marcusnaslund) [inline source/draw/RasterImage.ooc]: There's a lot of reused code here, both cases are pretty identical but switch x and y, as far as I can see. The same goes for the first two cases. Maybe we can clean that up.

Comment (davidpiuva) [inline source/draw/RasterImage.ooc]: Anything I can think of would lead to performance loss.

**Comment by marcusnaslund** [comment]
Frozen awaiting `primaryFeature1.4` merge.

File: source/draw/RasterImage.ooc [hunk 2]
```diff
@@ -66,17 +66,32 @@ RasterImage: abstract class extends Image {
 		} else {
 			originalPen := pen
```
Comment (marcusnaslund) [inline source/draw/RasterImage.ooc]: Is this line some residual of how `Pen` used to work? Looks like we can remove it.

**Comment by davidpiuva** [comment]
Updated

**Pull request merged**